Automattic/wp-calypso
# Composite checkout: cleanup redundant components

#### Changes proposed in this Pull Request

This PR removes the `grid-row` component from the wp-com folder, reduces duplicate code of the CVV image, and fixes a regression in ie11 related to the CVV image. 

![image](https://user-images.githubusercontent.com/6981253/69351611-9af33f00-0c49-11ea-928c-d9c9ee4f157f.png)

#### Testing instructions
* Get checkout up and running on your local machine with these instructions: https://github.com/Automattic/wp-calypso/pull/37013.
* Review the CVV image in the credit cards and then make your way to the billing section to make sure everything looks good.
